src/components/calcite-value-list-item/calcite-value-list-item.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Listen,
+  Method,
+  Prop,
+  h
+} from "@stencil/core";
+import { ICON_TYPES } from "../calcite-pick-list/resources";
+
+@Component({
+  tag: "calcite-value-list-item",
+  styleUrl: "./calcite-value-list-item.scss",
+  shadow: true
+})
+export class CalciteValueListItem {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Prop() selected = false;
+
+  @Prop({ reflect: true }) editable = false;
```

Comment:
pebble: are we using `editable` anywhere for now?

Reply:
It gets added as an attr on the wrapped `pick-list-item` but doesn't seem to have any other effect.

Reply:
um....yea that's right we removed the label editing. Will remove this for now.

Reply:
👍